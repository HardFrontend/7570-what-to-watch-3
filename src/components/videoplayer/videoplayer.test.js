import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './videoplayer.jsx';

const movie = {
  title: `Bohemian rhapsody`,
  imgPoster: `img/bohemian-rhapsody.jpg`,
  genre: `Comedy`,
  releaseDate: 2020,
  imgBg: `img/bg-the-grand-budapest-hotel.jpg`,
  runTime: `1h 50m`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`, `Edward Norton`, `Jude Law`],
  video: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Render VideoPlayer`, () => {
  const tree = renderer
    .create(<VideoPlayer
      movie={movie}

    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});


