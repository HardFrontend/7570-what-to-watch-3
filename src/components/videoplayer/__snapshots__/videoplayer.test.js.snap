// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer 1`] = `
<video
  controls={true}
  onClick={[Function]}
  poster="img/bohemian-rhapsody.jpg"
  width="100%"
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  />
</video>
`;
